docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+* Generated queries should be as close as possible as the real queries sent by users, which means likely purposefully generating “bad” data. In my current instruction dataset, most queries have no punctuation, some are in all caps or missing words, with typos.
+* Generated answers have to convey the general “brand” style of your system. Do not hesitate to re-generate the answer, reformulate the existing one, or even rewrite them manually.
+
+Relatedly, you have to anticipate the “persona” of your model. Whether you like it or not, users of interactive systems will ask meta-referential questions: who are? What can you do? Many advanced LLM systems overlooked this aspect to their detriment (with new models trained from scratch erroneously stating they are ChatGPT, since this is the default AI persona in much of the training data these days). I recommend here to combine both prepared answers in the instruction dataset with a series of “biographical” documents about the model in the RAG dataset.
+
+To assess style and frictions, the best way may ultimately be to collect feedback from your users. You don’t need anything fancy: I would just recommend an open-ended comment section somewhere (and green/red buttons).
+
+# 5. Will RAG redefine LLMs?
+By mid-2024, we are probably at a crossroad. There's always the possibility of a major breakthrough in LLM research (Q*, Monte Carlo Tree Search, diffusion models, or state space models…) that could shift the focus back to model architecture. If not LLMs will be increasingly pushed to the backstage of larger, more complex systems.
+
+RAG has emerged as the main production use case for LLMs and will be at the forefront of model orchestration. Orchestration already happened in many ways. Embedding models are language models, and so are classifiers. Integrating multiple decoder models in production will just be a scaling challenge: the pattern is already there.
+
+We are now at the early stages of a new generation of LLMs and SLMs made for RAG and RAG-associated tasks. The first models are already there. CommandR from Cohere, has been designed from the ground up with better support for source quotation and document interaction. The smallest variant of Phi-3 from Microsoft aims to operate best in contexts where knowledge and memory are separated from reasoning capacity.
+
+In the current state of technology adoption, specialization is solving multiple complex equations. It reduces the overall memory footprint, as small models well integrated in the pipeline can suddenly become competitive with gigantic frontier models. It ensures the model can run locally if needs be and process private organizational data without any liability. It limits the amount of work needed to deploy the model in production as many requirements of a good RAG application (source verifiability, better grounding, better understanding of original sources) can be addressed at the pretraining level.
```

Comment:
"solving multiple complex equations" = ? you mean "complex problems" or specifically "equations"?